File: C/7 Kyu/vowel_count.c
#include <stddef.h>
#include <string.h>
size_t get_count(const char *s)
{
    size_t cont = 0;
    size_t tam  = strlen(s);
    for(size_t i = 0; i<tam; i++){
        
        if (s[i]=='a' || s[i]=='e' ||s[i]=='i' ||s[i]=='o' ||s[i]=='u' ||s[i]=='A' ||s[i]=='E' ||s[i]=='I' ||s[i]=='O' ||s[i]=='U'){
           cont++;
        }
    }
  return cont;
}

File: Rust/7 Kyu/vowel_count.rs
fn is_vowel(c: char) -> bool {
    match c {
        'a'  | 'e'  | 'i'  | 'o'  | 'u'  => true,
        _ => false,
    }
}



fn get_count(string: &str) -> usize {
    let mut cont: usize = 0;
    for x in string.chars(){
        if is_vowel(x){
            cont+=1;
        } 
    }
  // Do your magic here
  
  cont
}